src/annotator/integrations/html-side-by-side.js

```diff
@@ -0,0 +1,171 @@
+/**
```

Comment:
This file contains various helpers used to implement side-by-side mode in HTML and EPUB documents, which have been split out for ease of testing and to keep `HTMLIntegration`'s complexity more manageable.